src/library_postprocessing.rs

```diff
@@ -124,18 +124,26 @@ impl Library {
     }
 
     fn c_type_by_type_id(&self, tid: TypeId) -> Option<String> {
-        use library::Type::*;
         let type_ = self.type_(tid);
         let glib_name = type_.get_glib_name();
```

Comment:
Now I understand. Also after splitting out `is_referenced_type` this bit can become simpler:
```rust
type_.get_glib_name().map(|glib_name| {
...
```